js/admin/modules-position.js

```diff
@@ -165,6 +165,8 @@ $(function(){
 
 		var $this = $(this);
 		var hook_select = $("select[name='id_hook']");
+		var optgroup_unregistered = hook_select.find('optgroup#hooks_unregistered');
```

Comment:
Using the selector with the ID should me for performant. It will use the native method `getElementById()`.

```suggestion
		var optgroup_unregistered = $('#hooks_unregistered');
```

Reply:
Thanks, I made the change in https://github.com/PrestaShop/PrestaShop/pull/13033/commits/0f1a20e7f775c37373b6a7260b6e64c48e494538